tests/foreman/ui/test_docker.py

```diff
@@ -369,11 +654,35 @@ def test_publish_multiple_docker_repo_composite_content_view(self):
 
         @Feature: Docker
 
-        @Status: Manual
-
         """
+        repo_name = gen_string('alpha')
+        content_view = entities.ContentView(
+            composite=False,
+            organization=self.organization,
+        ).create()
+        with Session(self.browser) as session:
+            _create_repository(
+                session,
+                org=self.organization.name,
+                name=repo_name,
+                product=self.product.name,
+            )
+            self.assertIsNotNone(self.repository.search(repo_name))
+            self.navigator.go_to_content_views()
+            self.content_views.add_remove_repos(
+                content_view.name, [repo_name], repo_type='docker')
+            self.content_views.publish(content_view.name)
+
+            composite_name = gen_string('alpha')
+            self.content_views.create(composite_name, is_composite=True)
+            self.navigator.go_to_content_views()
+            self.content_views.add_remove_cv(
+                composite_name, [content_view.name])
+            for _ in range(randint(2, 5)):
```

Comment:
Following same approach as in CLI and API. Basically, it is just more coverage (like have one time 2 instances of entity and 4 in another)